smartsim/_core/utils/helpers.py

```diff
@@ -43,18 +43,35 @@
 from datetime import datetime
 from shutil import which
 
+from typing_extensions import TypeAlias
+
 if t.TYPE_CHECKING:
     from types import FrameType
 
     from typing_extensions import TypeVarTuple, Unpack
 
+    from smartsim.launchable.job import Job
+
     _Ts = TypeVarTuple("_Ts")
 
 
 _T = t.TypeVar("_T")
 _HashableT = t.TypeVar("_HashableT", bound=t.Hashable)
 _TSignalHandlerFn = t.Callable[[int, t.Optional["FrameType"]], object]
 
+_nested_seq: TypeAlias = "t.Sequence[Job | _nested_seq]"
```

Comment:
do you mind explaining this type? Idk why my brain is farting but its a sequence of Jobs or input that matches the _nested_seq type?